xml/FrameworksIndex/net-5.0.xml

```diff
@@ -44556,56 +44555,6 @@
       <Member Id="F:System.Runtime.InteropServices.ComTypes.VARKIND.VAR_STATIC" />
     </Type>
   </Namespace>
-  <Namespace Name="System.Runtime.InteropServices.WindowsRuntime">
```

Comment:
Not sure why these got deleted.

Reply:
It looks like this is expected. For example, EventRegistrationTokenTable was removed in [this commit](https://github.com/dotnet/runtime/pull/36715/commits/77a50e79ae2f1b0d93689e02f39e3912cdc20157?file-filters%5B%5D=).